packages/boot/test/bootstrapTests/vats-restart.test.ts

```diff
@@ -1,43 +1,47 @@
 /** @file Bootstrap test of restarting (almost) all vats */
-import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
 import type { TestFn } from 'ava';
 
-import { Fail } from '@endo/errors';
-import { eventLoopIteration } from '@agoric/internal/src/testing-utils.js';
+import { makeWalletFactoryDriver } from '@aglocal/boot/tools/drivers.js';
+import { makeMockBridgeKit } from '@agoric/cosmic-swingset/tools/test-bridge-utils.ts';
+import { makeCosmicSwingsetTestKit } from '@agoric/cosmic-swingset/tools/test-kit.js';
+import { buildProposal } from '@agoric/cosmic-swingset/tools/test-proposal-utils.ts';
 import { Offers } from '@agoric/inter-protocol/src/clientSupport.js';
-import { makeAgoricNamesRemotesFromFakeStorage } from '@agoric/vats/tools/board-utils.js';
+import { makeFakeStorageKit } from '@agoric/internal/src/storage-test-utils.js';
 import type { ScopedBridgeManager } from '@agoric/vats';
-import { makeSwingsetTestKit } from '../../tools/supports.js';
-import { makeWalletFactoryDriver } from '../../tools/drivers.js';
-
-// main/production config doesn't have initialPrice, upon which 'open vaults' depends
-const PLATFORM_CONFIG = '@agoric/vm-config/decentral-itest-vaults-config.json';
+import { makeAgoricNamesRemotesFromFakeStorage } from '@agoric/vats/tools/board-utils.js';
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { Fail } from '@endo/errors';
 
-export const makeTestContext = async t => {
+export const makeTestContext = async () => {
   console.time('DefaultTestContext');
-  const swingsetTestKit = await makeSwingsetTestKit(t.log, undefined, {
-    configSpecifier: PLATFORM_CONFIG,
+  const storage = makeFakeStorageKit('bootstrapTests');
+
+  const { handleBridgeSend } = makeMockBridgeKit({ storageKit: storage });
+  const swingsetTestKit = await makeCosmicSwingsetTestKit({
+    // main/production config doesn't have initialPrice, upon which 'open vaults' depends
+    configSpecifier: '@agoric/vm-config/decentral-itest-vaults-config.json',
```

Comment:
Opportunity for cleanup and better consistency:
```suggestion
    configSpecifier: '@agoric/vm-config/decentral-itest-vaults-config.json',
```